Calling Errors
===============

.. currentmodule:: pywa.errors

.. autoclass:: CallingError()
    :show-inheritance:

.. autoclass:: CallingNotEnabled()
    :show-inheritance:

.. autoclass:: ReceiverUncallable()
    :show-inheritance:

.. autoclass:: ConcurrentCallsLimit()
    :show-inheritance:

.. autoclass:: DuplicateCall()
    :show-inheritance:

.. autoclass:: CallConnectionError()
    :show-inheritance:

.. autoclass:: CallRateLimitExceeded()
    :show-inheritance:

.. autoclass:: CallPermissionNotFound()
    :show-inheritance:

.. autoclass:: CallConnectionTimeout()
    :show-inheritance:

.. autoclass:: CallPermissionRequestLimitHit()
    :show-inheritance:

.. autoclass:: BusinessInitiatedCallsLimitHit()
    :show-inheritance:

.. autoclass:: FetchCallPermissionLimitHit()
    :show-inheritance:
